### ISSUE TITLE
default "agent-name" has unnecessary quotes

### ISSUE THREAD

**Issue by matthewdarwin** [EOSIO/eos#8333]
According to:

```
nodeos --print-default-config | grep agent
# agent-name = "EOS Test Agent"
```
It would appear quotes are needed for the "agent-name".  Actually no quotes are needed as the the quotes actually appear as part of the handshake.  So really I would expect to see no quotes in the output:

```
nodeos --print-default-config | grep agent
# agent-name = EOS Test Agent
```

**Comment by matthewdarwin** [comment]
Maybe a better default would be to have an empty string?

**Comment by jgiszczak** [comment]
Resolved by #8345.
